sdk/healthinsights/azure-healthinsights-cancerprofiling/setup.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+# coding: utf-8
+
+import os
+import re
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "azure-healthinsights-cancerprofiling"
+PACKAGE_PPRINT_NAME = "None"
```

Comment:
None?

Reply:
fixed to :
"Cognitive Services Health Insights Cancer Profiling"
and
in clinical matching to "Cognitive Services Health Insights Clinical Matching"